src/slab.js

```diff
@@ -127,6 +127,9 @@ async function waitForInstance(taskId, taskName) {
       const task_status = body[taskName].status.toLowerCase()
 
       if (task_status === 'done') {
+        if (taskName === 'start') {
+          await acknowledgeTaskDone(taskId)
+        }
```

Comment:
did we store the runner name at this point ?

Reply:
Yes. It's done right after start request.